### DISCUSSION TITLE
Terraform

### DISCUSSION THREAD

**Discussion by keichigo** [community/community#65854] [New to GitHub] [answered]
Hi Team, I want to write code in terraform to spin up github repo. I need github version, but could find this github version. Can you please help me.

Please find the below piece of code and please provide the version of this github account.
terraform {
required_providers {
github = {
source = "integration/github"
version = "?"
}
}
}

**Comment by mehulrajdev** [comment, marked as answer]
Hi! The latest version of Github on TF appears to be 5.34.0, which you can find in the TF docs at https://registry.terraform.io/providers/integrations/github/latest/docs
